docs/contributing/authoring-polyfills.html

```diff
@@ -0,0 +1,173 @@
+{{>header}}
+
+{{>nav this}}
+
+	<div class="o-techdocs-main o-techdocs-content">
+
+		<h1>Authoring polyfills</h1>
+
+		<p>Polyfills are located in the <code>/polyfills</code> directory and organised by feature name.</p>
```

Comment:
Would be useful to know what to call the polyfill, example being that when I did the picture polyfill pull request you asked me to move it to HTMLPictureElement, so something that suggests the folder/polyfill name might be useful